build-tools/automation/azure-pipelines.yaml

```diff
@@ -279,7 +279,7 @@ stages:
         testResultsFiles: TestResult-*.xml
         testRunTitle: Java Interop Tests - Windows Build Tree
 
-    - script: dotnet tool update apkdiff -g
+    - script: dotnet tool update apkdiff --version 0.0.13 --add-source https://api.nuget.org/v3/index.json -g
       displayName: install apkdiff dotnet tool
       continueOnError: true
```

Comment:
Should we make a `install-apkdiff.yaml` template?

Then we'd have the version in one place, too.

Reply:
Good call, done.